guides/v2.1/cloud/basic-information/starter-develop-deploy-workflow.md

```diff
@@ -139,60 +117,66 @@ The generated file is located in `app/etc/`:
 * For 2.1.4 and later: `config.local.php`
 * For 2.2 and later: `config.php`
 
-You will generate the file in the Integration environment where you configured Magento. We walk you through the process of generating the file, adding it to your Git branch, and deploying it.
+The generated file is `app/etc/config.php`.
+
+You generate the file in the Integration environment where you configured Magento. We walk you through the process of generating the file, adding it to your branch, and deploying it.
 
 **Important notes** on Configuration Management:
 
-* Any configuration setting included in the file is locked from editing, or read-only, in the deployed environment. This is one reason we recommend using `scd-dump`.
+* Any configuration setting included in the file generated from the `app:config:dump` command is locked from editing, or read-only, in the deployed environment. This is one reason we recommend using `scd-dump`.
```

Comment:
`scd-dump` may be not very clear. People try using it from regular bin/magento
I would suggest to write whole command